src/net/quic/send_posix.zig

```diff
@@ -0,0 +1,56 @@
+const std = @import("std");
```

Comment:
would pref to not have nested folders if we can avoid it -- in this case i think both these files can just remain in net/ -- would probs make sense to rename `client.zig -> quic.zig` and `send_posix.zig -> send_msg.zig` and then have a `quic_demo.zig` or `quic_main.zig` which has the main() method for the binary/cmd.zig entrypoint

Reply:
done in [25afde4](https://github.com/Syndica/sig/pull/314/commits/25afde469b5031d47f461c19af5cc1aa75f414c3)